cmd/clusterctl/client/config.go

```diff
@@ -134,7 +134,7 @@ type GetClusterTemplateOptions struct {
 	ClusterName string
 
 	// KubernetesVersion to use for the workload cluster. If unspecified, the value from os env variables
-	// or the .cluster-api/clusterctl.yaml config file will be used.
+	// or the $XDG_CONFIG_HOME/clusterctl.yaml or .cluster-api/clusterctl.yaml config file will be used.
```

Comment:
should this be `XDG_CONFIG_HOME/cluster-api/clusterctl.yaml` ?

Reply:
Yeah, you're right 👍️